lib/improver/metadata/utilities.py

```diff
@@ -70,5 +66,15 @@ def create_coordinate_hash(cube):
         coordinate_hash (str):
             A hash created using the x and y coordinates of the input cube.
     """
-    hashable_data = [cube.coord(axis='x'), cube.coord(axis='y')]
+    hashable_data = []
```

Comment:
I think this is a good change, since it moves the data out of Iris specific types into standard python types and makes it clear what metadata will be included in the hash.